ngraph_bridge/ie_executable.cc

```diff
@@ -91,59 +91,56 @@ IE_Executable::IE_Executable(shared_ptr<Function> func, string device)
   NGRAPH_VLOG(2) << "Loading IE CNN network to device " << m_device;
 
   InferenceEngine::Core ie;
-  // Load model to the plugin (m_device)
+  // Load network to the plugin (m_device) and create an infer request
   InferenceEngine::ExecutableNetwork exe_network =
       ie.LoadNetwork(m_network, m_device);
-
-  // Create infer request
   m_infer_req = exe_network.CreateInferRequest();
 }
 
 bool IE_Executable::call(const vector<shared_ptr<runtime::Tensor>>& outputs,
                          const vector<shared_ptr<runtime::Tensor>>& inputs) {
   InferenceEngine::InputsDataMap input_info = m_network.getInputsInfo();
-  if (input_info.size() != (inputs.size() + m_hoisted_params.size())) {
+  if (input_info.size() != (inputs.size() + m_inputs.size())) {
```

Comment:
This check looks a bit confusing. May be some comment will help.

Reply:
Yeah, agreed. Will add a comment here.